gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -272,13 +277,36 @@ message TripUpdate {
       // NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
       optional string assigned_stop_id = 1;
 
+      // The updated headsign of the vehicle at the stop.
+      optional string stop_headsign = 2;
+
+      // The updated pickup of the vehicle at the stop.
+      optional DropOffPickupType pickup_type = 3;
+
+      // The updated drop off of the vehicle at the stop.
+      optional DropOffPickupType drop_off_type = 4;
```

Comment:
Are these three fields only intended for ADDED / REPLACEMENT trips?

Reply:
I currently only specify them for ADDED / REPLACEMENT trips as there has been no consensus if I should be allowed to modify them for SCHEDULED trips or not.